### PR TITLE
Add tip to link to command for creating directories

### PR DESCRIPTION
**Pull request by prees1**
Adds a tip to link to the command that can be used to run source code analysis on lots of applications.

### CODE REVIEW

**Review by jmariomejiap — APPROVED** [review]
Nice, Thanks for adding this 👍

**Comment by prees1** [comment]
great thanks. Will merge once this PR is reviewed and merged https://github.com/tidalmigrations/gists/pull/5

**Pull request merged**
